api_docs/changelog.md

```diff
@@ -18,6 +18,18 @@ clients should check the `zulip_feature_level` field, present in the
 /register`](/api/register-queue) responses, to determine the API
 format used by the Zulip server that they are interacting with.
 
+## Changes in Zulip 7.1
+
+**Feature level 184**
```

Comment:
This seems to have changed with the rebase.

Reply:
This feature may not be released in `7.0`.

Reply:
Adding the fix.